packages/dds/test-dds-utils/src/ddsFuzzHarness.ts

```diff
@@ -195,6 +197,8 @@ export interface DDSFuzzModel<
 		channelA: ReturnType<TChannelFactory["create"]>,
 		channelB: ReturnType<TChannelFactory["create"]>,
 	) => void;
+
+	minimizationTransforms?: MinimizationTransform<TOperation>[];
```

Comment:
Could you write some doc here & on `MinimizationTransform` type?

It'd be nice to write tests for this too at some point, but seems fine to check in and build from there: this seems like a solid value add as-is.